docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -95,8 +95,7 @@ OpenSearch Python client offers a helper called bulk() which allows us to send m
 
 Get data mapping with Python
 ----------------------------
-When we sent the data to OpenSearch cluster on :ref:`load the data with Python <load-data-with-python>`, no data structure was specified. 
-As default, OpenSearch uses **dynamic mapping** to automatically add the fields to their type mapping. 
+When we sent the data to OpenSearch cluster on :ref:`load the data with Python <load-data-with-python>`, no data structure was specified. OpenSearch uses **dynamic mapping** to automatically add the fields to their type mapping when no data structure is specified which is our case.
```

Comment:
We repeat information and it reads a bit heavy to me, tbh.

Why not to be shorter - "When no data structure is specified, like in our case, OpenSearch uses **dynamic mapping** to automatically detect the fields."